exercise-1/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html,
+body {
+  height: 100vh;
+  margin: 0;
+  font-family: 'Lato', 'san-serif';
+}
+
+button {
+  display: none;
+}
+
+
+nav {
+  min-width: 150px;
+  display: flex;
+  justify-content: end;
+  height: 100vh;
+  color: #fff;
+  text-align: center;
+  background-color: #b2a68d;
+}
+
+nav a {
+  font-size: 16px;
+  color: #fff;
+  display: block;
+  padding: 12px;
+  padding-left: 30px;
+  transition: background-color 0.3s ease-in-out;
```

Comment:
When you transition `background-color` here, but not `color` and `border-radius`, it creates a flickering effect on hover. Try transitioning every attribute that changes value on hover.